test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -779,23 +779,47 @@ void codecFuzz(const test::common::http::CodecImplFuzzTestCase& input, HttpVersi
   server_connection.dispatcher_.to_delete_.clear();
 }
 
+#ifdef FUZZ_PROTOCOL_http1
+void codecFuzzHttp1(const test::common::http::CodecImplFuzzTestCase& input) {
+  codecFuzz(input, HttpVersion::Http1);
+}
+#endif
+
+#ifdef FUZZ_PROTOCOL_http2
+void codecFuzzHttp2Nghttp2(const test::common::http::CodecImplFuzzTestCase& input) {
+  codecFuzz(input, HttpVersion::Http2Nghttp2);
+}
+
+void codecFuzzHttp2WrappedNghttp2(const test::common::http::CodecImplFuzzTestCase& input) {
+  codecFuzz(input, HttpVersion::Http2WrappedNghttp2);
+}
+
+void codecFuzzHttp2Oghttp2(const test::common::http::CodecImplFuzzTestCase& input) {
+  codecFuzz(input, HttpVersion::Http2Oghttp2);
+}
+#endif
+
 } // namespace
 
 // Fuzz the H1/H2 codec implementations.
 DEFINE_PROTO_FUZZER(const test::common::http::CodecImplFuzzTestCase& input) {
   try {
     // Validate input early.
     TestUtility::validate(input);
-    codecFuzz(input, HttpVersion::Http1);
-    codecFuzz(input, HttpVersion::Http2Nghttp2);
-    codecFuzz(input, HttpVersion::Http2WrappedNghttp2);
+#ifdef FUZZ_PROTOCOL_http1
+    codecFuzzHttp1(input);
+#endif
+#ifdef FUZZ_PROTOCOL_http2
```

Comment:
nit: add a small comment explaining that we have different function names so they will appear explicitly in the stacktrace.

Reply:
done